kwalify/certification/v1.0.0.yaml

```diff
@@ -0,0 +1,12 @@
+type: map
+mapping:
+  standards:
+    type: map
+    mapping:
+      =:
```

Comment:
I don't see how you could do this without the =: wildcard match. Certifications contain the name of a standard under the 'standards' node and this needs to be matched.